ajax/libs/qwest/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "qwest",
+  "filename": "qwest.min.js",
+  "description": "Ajax library with XHR2, promises and request limit",
+  "version": "4.4.2",
+  "author": "Aurélien Delogu <pyrsmk@dreamysource.fr> (http://dreamysource.fr)",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pyrsmk/qwest.git"
+  },
+  "keywords": [
+    "ajax",
+    "request",
+    "promises",
+    "xhr"
+  ],
+  "license": "MIT",
+  "homepage": "http://dreamysource.fr/product/qwest",
```

Comment:
The website is just not here yet, but in development. Anyway, the future correct URL will be `http://dreamysource.io/project/qwest` instead of `http://dreamysource.fr/product/qwest`